frontend/packages/dev-console/integration-tests/support/pages/functions/createGitWorkload.ts

```diff
@@ -11,6 +11,7 @@ export const createGitWorkload = (
   appName: string = 'nodejs-ex-git-app',
   isPipelineSelected: boolean = false,
 ) => {
+  navigateTo(devNavigationMenu.Add);
```

Comment:
I recommend not to update add new action to existing functions, it might break other scripts or increase the execution time, feel free to add this line under step definitions.